### DISCUSSION TITLE
Beginner request

### DISCUSSION THREAD

**Discussion by cyy-2024** [py-pdf/pdfly#70] [Show and tell]
I'm a beginner who wants to contribute to the community, can I help with something?

**Comment by Lucas-C** [comment]
Hi @cyy-2024 👋

Welcome!

Sorry I missed your message last week.

Sure, you are very welcome to give a hand!

Are you a beginner Python developper?
You could have a look at our issues labelled [good first issue](https://github.com/py-pdf/pdfly/labels/good%20first%20issue)

And if you are not inspired by any of them, we can surely suggest other things to do 🙂

I'm converting this issue into a discussion.

**Reply by cyy-2024** [reply]
thank you very much~
